master/reference/node/configuration.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Configuring calico/node
+---
+
+The `calico/node` container is primarily configured through environment variables.
+
+
+## Environment Variables
+
+| Environment | Description                              | Schema | Default |
+| ------------- | -------- | ------ | ----- |
+| NODENAME | A unique identifier for this host. | string | |
+| NO_DEFAULT_POOLS | Prevents Calico from creating a default pool if one does not exist. | string | |
+| HOSTNAME [Deprecated] | The Hostname of this host. This is used as a unique identifier for the node. This value is overridden by NODENAME. | string | |
+| IP | The IP address for Calico to bind to. When specified, the address is saved in the [node resource configuration]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/node) for this host, overriding any previously configured value. When omitted, if an address **has** been saved in the node resource, then that value will be used. When omitted, if an address **has not** yet been configured in the node resource, the node will auto-detect an IPv4 address and configure the node resource with that address. This autodetection can be forced (even if a value has already been set in the node resource) by setting IP to "autodetect". Doing so will overwrite any value configured in the node resource. | ip address | |
```

Comment:
@caseydavenport we spoke about this, one thing I suggested was a table of all of the environments with a single line summary of what it is, followed by a section of heading/description -- so that way we could pad out the details without filling up a table.

We figured having all the information in a table would be a good increment - it wasn't clear initially if we'd need it.